projects/soft_999_eddy_kim_fw_102/src/main.c

```diff
@@ -0,0 +1,44 @@
+#include <stdint.h>
+#include <stdio.h>
+
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
+#define COUNTER_A_WAIT_TIME 500
+
+void prv_timer_callback(SoftTimerId timer_id, void *context) {
```

Comment:
Since this is a private function, it should be marked `static`, which makes it local to the file---otherwise, if another file tried to define a `prv_timer_callback`, you might get linker errors.